docs/versionsdescription.rst

```diff
@@ -1,44 +1,81 @@
 .. _versions-description:
 
-AGC Versions
+Versions
```

Comment:
Maybe "AGC analysis task versions"? These are essentially the relevant versions for other implementations to work against.